### ISSUE TITLE
Can't select template: no linked data

### ISSUE THREAD

**Issue by sedubois** [relax/relax#320]
- create a `Homepage` with a Section containing a `Hello world!` TextBox, save the changes
- create a `Main Navigation` and drag the `Homepage`, save the changes
- create a `Default Template` and add the `Main Navigation`
- change the `Main Navigation` background color so that you can see it (otherwise for some reason, both background and text are white). Save the changes.
- go back to `Homepage`, click Template in top right: right panel opens with `Default Template` visible
- but it's not selectable, has description "No linked data"

**Comment by bruno12mota** [comment]
You need to link something on the template. Sorry if that's still a bit unclear. With some tutorial videos that should be really simple to understand, but on the template you need to link some property so that the template "knows" where to put the page data. Got what I meant? 😅 

![screen shot 2016-10-13 at 4 51 08 pm](https://cloud.githubusercontent.com/assets/4086031/19353937/487622cc-9165-11e6-80d5-19c599c5ada7.png)

For example in the image below, I'm editing a template, and for the page I want its content to show up in that container. (also you need to save the template after)

There's still work to do to make this clearer but any feedback is really awesome at this point

**Comment by sedubois** [comment]
Thanks 😄 OK so I also understood https://github.com/relax/relax/issues/331. So I created a template which has a container which specifies it should display content:

<img width="1246" alt="screen shot 2016-10-13 at 20 07 39" src="https://cloud.githubusercontent.com/assets/4217871/19359025/f6833e0c-9180-11e6-9c7d-8a0f7164368f.png">

Then I made sure to click save in top-right, and went to the Homepage settings, selected the template, clicked save, and reloaded http://localhost:8080/, but I still see only the Homepage contents, not the template around it. I tried things like unselecting then reselecting the template, and also unselecting/reselecting the Frontpage in the general settings, but didn't seem to help.
